Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,55 @@
+
+## CONSTRAINTS ON THE SUBSYSTEM
+The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner.  The constraints given below are generalized to allow for comparison of any potential contender for the microcontroller.  Constraints on the subsystem are as follows:
+1. The main job of the microcontroller will be to manage and operate the other subsystems to work cooperatively and synchronously.  The board to be chosen must support fairly sophisticated code and be powerful enough to head all of the other subsystems.
+2. The microcontroller board should not be a hefty price for the user budget.  With a budget of $300, a super-powered board is not affordable.  The board price ranges between $20-$60.  This range is gathered from estimating the costs of other components and reasoning where money can be saved.
+3. The microcontroller must support communication between components.  To do this, Serial Peripheral Interface (SPI), Universal Asynchronous Receiver/Transmitter (UART), or some form of hardware communication will need to be achievable on the board.  SPI is a synchronous method while UART is an asynchronous method for two devices connected over a short distance to liaise.
+4. External storage support is required as per the shall statements.  The microcontroller will need to provide it or allow for it in some form.  Universal Serial Bus (USB) is the preferred form of external storage.   
+5. A major goal of the project is to allow for synchronous operations of at least 3 air quality sensors.  Sensors may be either analog or digital, so there must be support for both.  The board must have enough Input/Output (I/O) pins to operate any combination of 3 sensors whether they be digital or analog.   Different sensors interface with a potentially different number of pins.  Preliminary research shows that a sensor may take up to 5 pins to operate.  Other sensors may have more, but it is more common for 5 and under.  This means the board must have at least 15 digital input pins and 15 analog input pins.
+6. The cpu clock speed does not need to be very fast.  The code will be mostly Input/Output heavy with less computation.  This means that there will be a lot of waiting for responses and does not require a high-speed processor.  A cpu clock speed of at least 1 MHz is preferred for fast bootup.  
+7. There must be sufficient memory for the code.  It is unknown how much memory the code will occupy.  Operating code with libraries and compilation could be upwards of or slightly over 150 KB in Flash/Disk/Storage..  To account for the worst case, the board will need over 150 KB.
```

Comment:
This is wishy-washy. First it is stated that memory requirements are unknown and later it is stated that 150 kB is adequate. Also, check on correct units for memory size (what is a "K"? - hint: there's an IEEE standard which covers this).